vnext/Microsoft.ReactNative.Managed/Microsoft.ReactNative.Managed.csproj

```diff
@@ -21,6 +21,7 @@
     <DocumentationFile>bin\$(Platform)\$(Configuration)\Microsoft.ReactNative.Managed.XML</DocumentationFile>
     <Deterministic>true</Deterministic>
     <ProduceReferenceAssembly>true</ProduceReferenceAssembly>
+    <ProduceReferenceAssemblyInOutDir>true</ProduceReferenceAssemblyInOutDir>
```

Comment:
What does this achieve?

Reply:
Starting in VS 2022, the ref assembly for managed projects is no longer produced in the output directory for release builds (instead it's left in the intermediates), which causes the nuget package creation to fail later because the file is missing from the build artifacts. Setting this property returns to the previous behavior. See https://learn.microsoft.com/en-us/dotnet/core/compatibility/sdk/6.0/write-reference-assemblies-to-obj